.github/workflows/gradle-tests.yml

```diff
@@ -51,9 +51,9 @@ jobs:
         run: ./gradlew --no-daemon --parallel clean spotlessCheck
 
   # ==================================================================
-  # Unit Tests
+  # Unit Tests for London
   # ==================================================================
-  unit-tests:
+  unit-tests-london:
```

Comment:
I wonder if its possible write a reusable action here to avoid duplicating everything, especially as some of the settings for memory usage etc, are tricky?